test.js

```diff
@@ -6,13 +6,49 @@
 var leftPad = require("./");
 var test = require("tape");
 
-test('left pad', function (assert) {
-  assert.plan(7);
-  assert.strictEqual(leftPad('foo', 5), '  foo');
+test('edge cases', function (assert) {
+  assert.plan(12);
   assert.strictEqual(leftPad('foobar', 6), 'foobar');
+  assert.strictEqual(leftPad('foobar', 5), 'foobar');
+  assert.strictEqual(leftPad('foobar', -1), 'foobar');
+  assert.strictEqual(leftPad('foobar', 6, '1'), 'foobar');
+  assert.strictEqual(leftPad('foobar', 5, '1'), 'foobar');
+  assert.strictEqual(leftPad('foobar', -1, '1'), 'foobar');
+  assert.strictEqual(leftPad('foobar', 8, ''), '  foobar');
+  assert.strictEqual(leftPad('foobar', 8, false), '  foobar', 'false default to space');
+  //assert.strictEqual(leftPad('foobar', 8, []), '  foobar', '[] default to space');
+  //assert.strictEqual(leftPad('foobar', 8, {}), '  foobar', '{} default to space');
```

Comment:
yeah, I know what the current behaviour is. Passing `[]` will result in nothing being padded, and `{}` with `[object Object]` being padded. They don't look right, but I am just not sure what the desired behaviour should be.

I prefer to handle them in a separate PR. We could enable these two tests at that time.